docs/commands.md

```diff
@@ -199,6 +203,9 @@ Available filters:
 |valor|`!channel raid moltres valor`| Tracks moltres raids where controlling team is valor|
 |harmony|`!channel raid venusaur harmony`| Tracks venusaur raids in uncontested gyms|
 
+###### !unraid
+
+`!channel unraid snorlax` Removes tracking of Snorlax raids. 
```

Comment:
add also the way to remove tracking per raid tier: "`!channel unraid level5` Removes tracking of tier 5 raids"